src/scittle/nrepl.cljs

```diff
@@ -31,9 +31,12 @@
     :complete (let [completions (completions (assoc msg :ctx @!sci-ctx))]
                 (nrepl-reply msg completions))))
 
-(when (.-SCITTLE_NREPL_WEBSOCKET_PORT js/window)
+(defn ws-url [host port path]
+  (str "ws://" host ":" port "/" path))
+
+(let [ws-port (or (.-SCITTLE_NREPL_WEBSOCKET_PORT js/window) 1340)]
```

Comment:
The creation of the websocket should be made optional. It should not happen in production and now it is always happening.